### PR TITLE
Add circom/snarkjs compatible output for r1cs and witness

### PR DESCRIPTION
**Pull request by Schaeff**
Add optional snarkjs output for r1cs and witness so that snarkjs can be used alongside ZoKrates:
- compile with zokrates (generating a snarkjs r1cs)
- setup with snarkjs
- compute with zokrates (generating a snarkjs witness)
- proof generation with snarkjs

~~Not blocking but annoying: https://github.com/iden3/snarkjs/issues/152~~

Todo:
- [ ] Fix CLI flags

### CODE REVIEW

File: zokrates_cli/src/ops/compute_witness.rs [hunk 1]
```diff
@@ -38,6 +39,13 @@ pub fn subcommand() -> App<'static, 'static> {
         .takes_value(true)
         .required(false)
         .default_value(cli_constants::WITNESS_DEFAULT_PATH)
+    ).arg(Arg::with_name("witness")
```
Comment (dark64) [inline zokrates_cli/src/ops/compute_witness.rs]: I think this should be called `circom-witness` or something similar, as we also output the witness.

**Comment by xhliu** [comment]
Any update on this? Would be great to be compatible with circom/snarkjs.

**Comment by Schaeff** [comment]
> Any update on this? Would be great to be compatible with circom/snarkjs.

This will be out soon

**Review by dark64 — APPROVED** [review]
LGTM

**Pull request merged**